pandas/io/formats/format.py

```diff
@@ -1257,17 +1245,17 @@ def _format(x):
         fmt_values = []
         for i, v in enumerate(vals):
             if not is_float_type[i] and leading_space:
-                fmt_values.append(" {v}".format(v=_format(v)))
+                fmt_values.append(f" {_format(v)}")
             elif is_float_type[i]:
                 fmt_values.append(float_format(v))
             else:
                 if leading_space is False:
                     # False specifically, so that the default is
                     # to include a space if we get here.
-                    tpl = "{v}"
+                    tpl = f"{_format(v)}"
```

Comment:
this is a template so I _think_ better to leave as is. otherwise change the variable name.